stdlib/public/stubs/Stubs.cpp

```diff
@@ -232,8 +241,19 @@ extern "C" uint64_t swift_float80ToString(char *Buffer, size_t BufferLength,
 /// if an error occurred, or EOF was reached.
 SWIFT_RUNTIME_STDLIB_INTERFACE
 extern "C" ssize_t swift_stdlib_readLine_stdin(char **LinePtr) {
+#if defined(_MSC_VER)
+  if (std::cin.eof()) {
+    *LinePtr = nullptr;
+    return -1;
+  }
+  std::string Line;
+  std::getline(std::cin, Line);
+  *LinePtr = reinterpret_cast<char *>(std::malloc(Line.size() + 1));
```

Comment:
Did you mean to call `memcpy()` somewhere?

Also, you shouldn't need to `reinterpret_cast`.